plugin-compass/src/main/java/com/mapbox/maps/plugin/compass/CompassViewImpl.kt

```diff
@@ -50,8 +36,8 @@ open class CompassViewImpl : CompassView, AppCompatImageView {
     )?.let { compassImage = it }
 
     // Layout params
-    val screenDensity = context.resources.displayMetrics.density
-    layoutParams = FrameLayout.LayoutParams((48 * screenDensity).toInt(), (48 * screenDensity).toInt())
+    val compassViewSize = resources.getDimensionPixelSize(R.dimen.compass_view_size)
+    layoutParams = FrameLayout.LayoutParams(compassViewSize, compassViewSize)
```

Comment:
:+1:

Reply:
If some of customers would like to change compass size (for example for automotive maybe it is good to have it bigger) they can override this dimension and we don't need to update or modify our code.

Reply:
good point
Should we think about min size? If I set size to `0` I'll not see this element. Of course it's my fault as a developer, but maybe we should print some alert in console?